.circleci/config.yml

```diff
@@ -195,17 +169,14 @@ workflows:
   build_and_test:
     jobs:
       - compile
-      - compile-16
       - coverage
       - test-soccer:
           requires:
             - compile
       - test-python:
           requires:
             - compile
-      - pylint:
-          requires:
-            - compile-16
+      - pylint
```

Comment:
Should this require compile? I honestly never do this stuff so I'm not sure

Reply:
It previously did back on 16.04. But I'm pretty sure there's no reason for it, it doesn't rely on compiled binaries or anything. I think it may have just been ordering of tasks, so we don't run pylint until the end? But I'm not sure about that at all.

Reply:
It definitely shouldn't require the 18.04 compile if pylint is still broken on that release for us though.

Reply:
This was a hack. The `compile` target used a newer verison of ubuntu, which shipped a version of pylint which had bugs which caused a crash on our codebase. This was a hack to pin the pylint to the ubuntu 16 version. Assuming pylint dosen't crash (or maybe if you just want to get rid of it), it should be safe to remove :)